lib/controller.rb

```diff
@@ -54,10 +56,19 @@ class SlowFood < Sinatra::Base
     erb :index
   end
 
-  get '/auth/login' do
+  get '/auth/login.erb' do
```

Comment:
By putting this .erb here, you are saying that when the user goes to www.yourpage.com/auth/login.erb then they should see the login page. If instead you keep it as it was, you'll let them to go `www.yourpage.com/auth/login` - it's cleaner and definitely the standard.